udata/core/dataset/models.py

```diff
@@ -90,6 +90,19 @@
 MAX_DISTANCE = 2
 
 
+class JsonLdExtrasMixin(object):
+
+    @staticmethod
```

Comment:
I don't think having a static method is useful. A sinple helper function should do the job